lib/recomb_map.c

```diff
@@ -214,48 +194,39 @@ recomb_map_phys_to_discrete_genetic(recomb_map_t *self, double x, uint32_t *locu
 double
 recomb_map_genetic_to_phys(recomb_map_t *self, double genetic_x)
 {
```

Comment:
Ditto here, we want this function as the higher level APIs that we're supporting still provide it.